model/src/main/proto/topic.proto

```diff
@@ -205,7 +205,7 @@ message PromotedStory {
   LessonThumbnail lesson_thumbnail = 9;
 
   // The indicator to handle if story is completed.
```

Comment:
Perhaps more clear: ``Indicates whether the topic containing this story is 'learned' (meaning at least one story was completed).``

This might need line wrapping.

Reply:
Done.